config/routes.rb

```diff
@@ -72,89 +72,90 @@
       post '/carry-over' => 'carry_over#create', as: :carry_over
 
       scope '/personal-details' do
-        get '/' => 'personal_details/base#new', as: :personal_details_new
-        post '/' => 'personal_details/base#create', as: :personal_details_create
-        get '/edit' => 'personal_details/base#edit', as: :personal_details_edit
-        post '/edit' => 'personal_details/base#update', as: :personal_details_update
+        get '/' => 'personal_details/base#new', as: :personal_details
+        post '/' => 'personal_details/base#create'
+        get '/edit' => 'personal_details/base#edit', as: :edit_personal_details
+        patch '/edit' => 'personal_details/base#update'
         get '/nationalities' => 'personal_details/nationalities#new', as: :nationalities
         post '/nationalities' => 'personal_details/nationalities#create'
         get '/nationalities/edit' => 'personal_details/nationalities#edit', as: :edit_nationalities
-        post '/nationalities/edit' => 'personal_details/nationalities#update'
+        patch '/nationalities/edit' => 'personal_details/nationalities#update'
         get '/languages' => 'personal_details/languages#edit', as: :languages
         post '/languages' => 'personal_details/languages#update'
```

Comment:
Line 84 - should this be PATCH as well?

Reply:
Yes, you're right they should all be patches as they are all just attrs on the application form 🤦